improvement-ideas.md

```diff
@@ -0,0 +1,16 @@
+# List of possible improvements
+
+## General
+- Add a possibility to log the warnings (currently, they are printed only on errors).
+- Add tests.
+- Fix the used python version.
+- Fix all warnings about style, typo, unused variables, etc.
+- Type the code.
+
+## ArcGIS to Geostyler-style
+- For ArcGIS to Geostyler-style, convert unit from pt to px systematically.
+
```

Comment:
```suggestion
- Refactor the code in several files (instead of a single very long file)

```

Reply:
I think every ideas are worth to be written. And that's a good one. Thanks